src/frontend/src/pages/home/home.tsx

```diff
@@ -42,14 +42,31 @@ export const Home: React.FC = () => {
                 <Grid item xs={4}>
                     <Paper elevation={3}>
                         <List component="nav">
-                            <Link
-                                to="/view-challenge"
-                                style={{ textDecoration: "none" }}
-                            >
+                            <Link to="/view-challenge"style={{ textDecoration: "none" }}>
```

Comment:
```suggestion
                            <Link to="/view-challenge" style={{ textDecoration: "none" }}>
```
There should be a space here.